Driver/Driver/Scenes/LoginScreen/LoginViewContracts.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  LoginViewContracts.swift
```

Comment:
we should call it LoginProtocols. Contracts is a blockchain term also a backend term.